ompi/mpi/man/man3/MPI_Comm_spawn.md

```diff
@@ -0,0 +1,267 @@
+trailing spaces are stripped. MPI looks for the file first in the
+working directory of the spawning process.
+
+## The `argv` Argument
+
+`argv` is an array of strings containing arguments that are passed to
+the program. The first element of `argv` is the first argument passed to
+`command`, not, as is conventional in some contexts, the `command` itself.
+The argument list is terminated by NULL in C and an empty string in
+Fortran (note that it is the MPI application's responsibility to ensure
+that the last entry of the `argv` array is an empty string; the compiler
+will not automatically insert it). In Fortran, leading and trailing
+spaces are always stripped, so that a string consisting of all spaces is
+considered an empty string. The constant `MPI_ARGV_NULL` may be used in C
+and Fortran to indicate an empty argument list. In C, this constant is
+the same as NULL.
+
+In C, the `MPI_Comm_spawn` argument `argv` differs from the argv
+argument of `main` in two respects. First, it is shifted by one element.
+Specifically, argv[0] of `main` contains the name of the program
+(given by command). argv[1] of `main` corresponds to argv[0]
+in `MPI_Comm_spawn`, argv[2] of `main` to argv[1] of
+`MPI_Comm_spawn`, and so on. Second, `argv` of `MPI_Comm_spawn` must be
+null-terminated, so that its length can be determined. Passing an argv
+of `MPI_ARGV_NULL` to `MPI_Comm_spawn` results in `main` receiving `argc` of
+1 and an `argv` whose element 0 is the name of the program.
+
+## The `maxprocs` Argument
+
+Open MPI tries to spawn `maxprocs` processes. If it is unable to spawn
+`maxprocs` processes, it raises an error of class `MPI_ERR_SPAWN`. If MPI
+is able to spawn the specified number of processes, `MPI_Comm_spawn`
+returns successfully and the number of spawned processes, m, is given
+by the size of the remote group of intercomm.
+
+A spawn call with the default behavior is called hard. A spawn call for
+which fewer than `maxprocs` processes may be returned is called soft.
+
+The `info` Argument
+
+The `info` argument is an opaque handle of type `MPI_Info` in C and
+INTEGER in Fortran. It is a container for a number of user-specified
+(key,value) pairs. `key` and `value` are strings (null-terminated
+char in C, character() in Fortran). Routines to create and
+manipulate the `info` argument are described in Section 4.10 of the
+MPI-2 standard.
+
+For the SPAWN calls, `info` provides additional,
+implementation-dependent instructions to MPI and the runtime system on
+how to start processes. An application may pass `MPI_INFO_NULL` in C or
+Fortran. Portable programs not requiring detailed control over process
+locations should use `MPI_INFO_NULL.`
+
+The following keys for `info` are recognized in Open MPI. (The reserved
+values mentioned in Section 5.3.4 of the MPI-2 standard are not
+implemented.)
+
+|Key|Type|Description|
+|---|---- |-----------|
+|host|char *|Host on which the process should be|
```

Comment:
Sorry to be picky here, but could we put all the text for one cell on one line?  That would let Markdown render it in a single row with wrapped text lines.  I.e.:

```suggestion
|host|char *|Host on which the process should be spawned.  See the orte_host man page for an explanation of how this will be used.|
```

Reply:
I wasn't going to do it like that because then the lines would have to be longer than the 80 character limit for the markdown render to not be confused

Reply:
Oh wow, the renderer has a limit of 80-character lines?  Bummer!  Ok.

Reply:
I mean the one renderer which makes it viewable in the terminal. I was just going off earlier when you told me that open-mpi sticks to the 80 character line limit. I think parsedown is capable of more if you'd like me to ignore line length though.